microsite/data/plugins/should-i-deploy.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Should I Deploy Today?
+author: NandoRFS
+authorUrl: https://github.com/NandoRFS
+category: Humor
```

Comment:
TIL this category already exists - that's great :D

Reply:
Yes, in the company where I work we already use the plugin that is in this category 😄 . Just one more question, when will the changes be merged? 🤔